MC/bin/o2dpg_sim_workflow.py

```diff
@@ -1272,6 +1272,26 @@ def addQCPerTF(taskName, needs, readerCommand, configFilePath, objectsFile=''):
                 readerCommand='o2-global-track-cluster-reader --track-types "ITS" --cluster-types "ITS"',
                 configFilePath='json://${O2DPG_ROOT}/MC/config/QC/json/its-clusters-tracks-qc.json')
 
+     ### CPV
+     if isActive('CPV'):
+        addQCPerTF(taskName='CPVDigitsQC',
+                   needs=[getDigiTaskName("CPV")],
+                   readerCommand='o2-cpv-digit-reader-workflow',
+                   configFilePath='json://${O2DPG_ROOT}/MC/config/QC/json/cpv-digits-task.json')
+
+     if isActive('CPV'):
```

Comment:
This could be done in one single `if`

Reply:
In principle yes but these are 2 different steps: 1st - qc of digits - can be done right after digitization. 2nd -qc of clusters - can be done only after reconstruction finished.

Reply:
Ah, sorry. I got what you mean.